Makefile

```diff
@@ -1,63 +1,32 @@
-OUTPUT        = phpbrew.phar
 TARGET        = phpbrew
-MOVE          = mv -v
-MKDIR         = mkdir -p
-COPY          = cp -v
+SIGNATURE     = $(TARGET).asc
 SUDOCP        = sudo cp
 INSTALL_PATH  = /usr/local/bin
-PERMISSION    = chmod +x
 TEST          = phpunit
 
-.PHONY: build phar
+$(TARGET): vendor $(shell find bin/ shell/ src/ -type f) box.json.dist .git/HEAD
```

Comment:
is the `shell find` necessary here? Wouldn't it be enough to have `bin shell src`?

Reply:
why is `.git/HEAD` here?

Reply:
I actually copy/pasted this from the Infection's makefile.

> is the `shell find` necessary here? Wouldn't it be enough to have `bin shell src`?

In my understanding depending on the directories would not track dependencies in the underlying files since Make uses the mtime of the targets. Here, we don't care about the directories themselves but care about every single file inside them.

> why is `.git/HEAD` here?

It may be not relevant right now but would be relevant if something like [Ocramius/PackageVersions](https://github.com/Ocramius/PackageVersions) is used. It would consider the current branch/tag when generating the lock file.

Reply:
👌